### PR TITLE
is_autobid option

### PR DESCRIPTION
**Pull request by 13bzhang**
Hi! Sorry it's been a while since my last pull request to add the `is_autobid` option. I made the code less verbose this time. I tested it today and it works.

### CODE REVIEW

File: R/fb_adset.R [hunk 1]
```diff
@@ -20,7 +20,7 @@ fbad_create_adset <- function(fbacc,
                               name,
                               optimization_goal = c('NONE', 'APP_INSTALLS', 'CLICKS', 'ENGAGED_USERS', 'EXTERNAL', 'EVENT_RESPONSES', 'IMPRESSIONS', 'LINK_CLICKS', 'OFFER_CLAIMS', 'OFFSITE_CONVERSIONS', 'PAGE_ENGAGEMENT', 'PAGE_LIKES', 'POST_ENGAGEMENT', 'REACH', 'SOCIAL_IMPRESSIONS', 'VIDEO_VIEWS'),
                               billing_event = c('APP_INSTALLS', 'CLICKS', 'IMPRESSIONS', 'LINK_CLICKS', 'OFFER_CLAIMS', 'PAGE_LIKES', 'POST_ENGAGEMENT', 'VIDEO_VIEWS'),
-                              bid_amount,
+                              is_autobid, bid_amount,
```
Comment (daroczig) [inline R/fb_adset.R]: I think we should set `is_autobid` to `FALSE` by default, as if there's no default value, then L66 below will fail.

**Review by daroczig — CHANGES_REQUESTED** [review]
Thanks a lot for this! I'd be happy to merge after the above small suggestion, then I can run roxygenize etc, don't worry about that part.

File: R/fb_adset.R (hunk 1 above)
Comment (13bzhang) [reply R/fb_adset.R]: OK. I will change that.

Comment (13bzhang) [reply R/fb_adset.R]: Also I need to add `is_autobid` to the documentation.

**Comment by 13bzhang** [comment]
@daroczig Hi, I made the changes just now.

**Review by daroczig — APPROVED** [review]

**Comment by daroczig** [comment]
Thanks a lot! Run `roxygen` at https://github.com/cardcorp/fbRads/commit/6d4b0ea75ffc5d3946778b8767ba55bb26933d78

**Pull request merged**